Libraries/PeriphDrivers/Source/WDT/wdt_me18.c

```diff
@@ -153,3 +153,36 @@ void MXC_WDT_ClearIntFlag(mxc_wdt_regs_t *wdt)
 {
     MXC_WDT_RevB_ClearIntFlag((mxc_wdt_revb_regs_t *)wdt);
 }
+
+int MXC_WDT_SetClockSource(mxc_wdt_regs_t *wdt, mxc_wdt_clock_t clock_source)
+{
+    const uint8_t clock_source_num = 8;
```

Comment:
Why's `clock_source_num` always 8?

Reply:
WDT supports 8 different clock sources but mostly not using all of 8 options. I set `clock_source_num` parameter and `clock_sources` array size to 8 because upcoming revisions can use other options which are not used now. To reduce code size, I can change them to number of supported clock source.